examples/NotificationMarker.pulsing.jsx

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { NotificationMarker, IconButton, Text } from '@itwin/itwinui-react';
+import { SvgNotification } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <div className='demo-container'>
+      <IconButton styleType='borderless'>
```

Comment:
The `IconButton`s in these examples need the `label` prop to be defined in order to pass the a11y tests.

Reply:
Ohh, thanks, added labels (https://github.com/iTwin/iTwinUI/pull/1999/commits/fc036834694f7da1602d4d952ac050ee4ec1015e).